arduino/WandCore/WandCore.ino

```diff
@@ -163,7 +168,7 @@ void setup()
       }
     })
     .onError([](ota_error_t error) {
-      debugE("OTA Error[%d]: ", error);
+      debugE("OTA Error[%s]: ", error);

```

Comment:
I haven't looked, but I'm guessing the type `ota_error_t` isn't a string, and your previous `%d` was probably more valid. Generally these error types are enumerations (like you check against below) and are often unsigned ints.